apps/indexer/docker/load-extensions.sh

```
@@ -1,6 +1,6 @@
-
-#!/bin/sh
-
-psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<EOF
-CREATE EXTENSION IF NOT EXISTS btree_gist;
+
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)